docs/docs/node-apis.md

```diff
@@ -26,8 +26,8 @@ exports.createPages = (_, pluginOptions, cb) => {
 }
 ```
 
-If your plugin does not do async work, you can just return directly.
+যদি আপনার প্লাগইন অ্যাসিঙ্ক কাজ না করে তবে আপনি সরাসরি সরাসরি ফিরে আসতে পারেন।
 
-## Usage
+## ব্যবহার
 
-Implement any of these APIs by exporting them from a file named `gatsby-node.js` in the root of your project.
+নামের একটি ফাইল থেকে এগুলি রফতানি করে এই কোনও এপিআই প্রয়োগ করুন`gatsby-node.js` আপনার প্রকল্পের মূল মধ্যে।
```

Comment:
Also this line needs to be re-translated as @nutboltu vai suggested already. It is not really making much sense now.

Reply:
I'm unable to figure out any other way of translating this line.  Any help or suggestion would be welcomed :) Except this, all above issues have been updated.

Reply:
```suggestion
এই API গুলোর যেকোনোটিই আপনি `gatsby-node.js` নামের ফাইল থেকে এক্সপোর্ট করে আপনার রুট প্রজেক্টে ইমপ্লিমেন্ট করতে পারেন।
```